cypress/support/auth-provider-commands/auth0.ts

```diff
@@ -1,53 +1,37 @@
 // @ts-check
 ///<reference path="../../global.d.ts" />
 
-import * as jwt from "jsonwebtoken";
-
-Cypress.Commands.add("loginByAuth0Api", (username: string, password?: string) => {
+// Note: this function leaves you on a blank page, so you must call cy.visit()
+// afterwards, before continuing with your test.
+Cypress.Commands.add("loginByAuth0Api", (username: string, password: string) => {
```

Comment:
Wonder if we should go with something generic here since we aren't leveraging the API directly any longer
```suggestion
Cypress.Commands.add("loginToAuth0", (username: string, password: string) => {
```

Reply:
Oh damn well spotted!